updater/lib/dependabot/api_client.rb

```diff
@@ -330,10 +330,13 @@ def record_ecosystem_meta(ecosystem)
     def version_manager_json(version_manager)
       return nil unless version_manager
 
+      raw_version = version_manager.version&.to_semver.to_s
+      version = version_manager.version&.to_semver.to_s
+
       {
         name: version_manager.name,
-        raw_version: version_manager.version.to_semver.to_s,
-        version: version_manager.version.to_s,
+        raw_version: raw_version.empty? ? "N/A" : raw_version,
+        version: version.empty? ? "N/A" : version,
```

Comment:
**Review Tip**: We want to be able to send empty versions if package manager doesn't have version.